server/controllers/entities/lib/resolver/update_resolved_entry.js

```diff
@@ -35,13 +35,23 @@ const getEntity = (prefix, entityId) => {
   }
 }
 
-const addMissingClaims = async (entity, seedClaims, imageUrl, reqUserId, batchId) => {
-  // Do not update if property already exists
+const updateClaims = async (entity, seedClaims, imageUrl, reqUserId, batchId) => {
+  // Do not update if property already exists (except if date is more precise)
   // Known cases: avoid updating authors who are actually edition translators
+  const updatedEntity = _.cloneDeep(entity)
   const newClaims = _.omit(seedClaims, Object.keys(entity.claims))
+  _.forEach(newClaims, (newValues, prop) => {
```

Comment:
[minor point: syntax]
I would be of the opinion that we should stick to standard JS options when available, rather than libs
```suggestion
  Object.keys(newClaims).forEach(prop => {
```
or
```suggestion
  for (const prop in newClaims) {
```

Reply:
I dont get it, to me, `2021-01` is more precise than `2020`, why would i be "better" to ignore this update ?

Reply:
this would add some significant changes in the code. What about seeing it as: if its more precise, most chances is that its more relevant (since its more difficult to make this up) ?